src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,77 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from '../../app/hooks';
+
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { actions as crntTodoActions } from '../../features/currentTodo';
```

Comment:
```suggestion
import { actions as currentTodoActions } from '../../features/currentTodo';
```
do not use short forms - it's better for readability